src/utils/_colors.scss

```diff
@@ -12,10 +12,16 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
 
+$color-variation-dark: black;
 $color-lighten-steps: 25, 50, 75, 90, 95 !default;
 $color-darken-steps: 20, 35, 55, 75 !default;
 
-@function color-variation($color, $step: 0, $dark: black, $light: white) {
+@function color-variation(
+  $color,
+  $step: 0,
+  $color-variation-dark,
+  $light: white
```

Comment:
wouldn't you change this name to "$color-variation-light" as well?

Reply:
yes! done